baseplate/lib/thrift_pool.py

```diff
@@ -95,6 +95,7 @@ def thrift_pool_from_config(
     parser = config.SpecParser(
         {
             "endpoint": config.Endpoint,
+            "fifo_queue": config.Optional(config.Boolean, default=False),
```

Comment:
Could you add this to the supported keys list above? Another option could be to accept a string defining what type of queue class to use in case more are added. Something like `"queue_cls": config.Optional(config.String, default="fifo"),` or if there's a way to specify a list of available options but I don't remember if that syntax is available atm

Reply:
I think the only options we really need to support here are LIFO and FIFO. We'd have to completely replace the scheduler if we wanted to do something more fancy.